### ISSUE TITLE
Create separate VMs for Safari and Edge runs

### ISSUE THREAD

**Issue by jeffcarp** [w3c/wptdashboard#137]
Currently Safari and Edge are run sequentially on instance-2. They should be run in their own VMs so all browsers start running at 12am PT (right now Edge generally begins around 4-5am). 

(via discussion on #testing)

**Comment by jeffcarp** [comment]
I created an instance `edge-windows-10-sauce`. Still in the process of verifying that it works, but once it's ready I'll turn `instance-2` into a Safari-only VM.

**Comment by jeffcarp** [comment]
cc/@bobholt 

- The instance `edge-windows-10-sauce-2` now exists and is set up to run Edge at 12am PT.
- I removed the `&& ./run/run.py edge` from `instance-2`.
  - Now `instance-2` is Safari only.
  - However the Edge run for Oct 7 will still run since the cron job with Safari & Edge began this morning and is still running.
- There were some modifications I had to make to `run.py` after moving to `wpt run`, I'll PR those.
